PowerEditor/src/ScitillaComponent/AutoCompletion.cpp

```diff
@@ -46,6 +46,48 @@ static bool isAllDigits(const generic_string &str)
 	return std::all_of(str.begin(), str.end(), ::isdigit);
 }
 
+bool AutoCompletion::addLanguageCustomChars()
+{
+	if (_funcCalltip._additionalWordChar == TEXT(""))
+		return false;
+
+	string charList;
+	auto defaultCharListLen = _pEditView->execute(SCI_GETWORDCHARS);
+	char *defaultCharList = new char[defaultCharListLen + 1];
+	_pEditView->execute(SCI_GETWORDCHARS, 0, reinterpret_cast<LPARAM>(defaultCharList));
+	charList = defaultCharList;

```

Comment:
This has a potential buffer overrun. See #3139 for a similar fix when using `SCI_GETWORDCHARS`.